101-workshop-nodejs/ex1-creating-and-invoking-actions/action-proxy.md

```diff
@@ -93,9 +93,9 @@ Let's look an example of creating a "proxy" action which invokes another action
     ```
 
 {% hint style="tip" %}
-**Note** On the invoke call above, we used the short form for the `--last` flag which is `-l` with a parameter to only `list` the last `2` activations.
+On the invoke call above, you used the short form for the `--last` flag which is `-l` with a parameter to only `list` the last `2` activations.
 {% endhint %}
 
 {% hint style="success" %}
-🎉**Congrats on proxying an action!** Be sure to _**[check out all the cool things](https://github.com/apache/openwhisk-client-js/#examples)**_ the [NPM OpenWhisk](https://www.npmjs.com/package/openwhisk) JavaScript library can be used for such _"invoking triggers"_ to fire events and _"chaining action calls"_ within actions.
```

Comment:
Can we preserve the hyperlink to the NPM package?

Reply:
It's in the information box under step 1, so that's why I removed it. I think it'd be better to pick one or the other so readers don't think they're two different links and click twice. But up to you!

Reply:
aha!